diff --git a/bazel/config/BUILD.bazel b/bazel/config/BUILD.bazel
index c81f2782..444b5adb 100644
--- a/bazel/config/BUILD.bazel
+++ b/bazel/config/BUILD.bazel
@@ -76,7 +76,7 @@ config_setting(
 
 config_setting(
     name = "platform_cpu_ppc64le",
-    constraint_values = ["@platforms//cpu:ppc"],
+    constraint_values = ["@platforms//cpu:ppc64le"],
 )
 
 v8_target_cpu(
diff --git a/bazel/defs.bzl b/bazel/defs.bzl
index 0539ea17..515ab7da 100644
--- a/bazel/defs.bzl
+++ b/bazel/defs.bzl
@@ -437,7 +437,7 @@ def _v8_target_cpu_transition_impl(settings,
         "armeabi-v7a": "arm32",
         "s390x": "s390x",
         "riscv64": "riscv64",
-        "ppc64": "ppc64le",
+        "ppc": "ppc64le",
     }
     v8_target_cpu = mapping[settings["//command_line_option:cpu"]]
     return {"@v8//bazel/config:v8_target_cpu": v8_target_cpu}
diff --git a/third_party/highway/BUILD.gn b/third_party/highway/BUILD.gn
index 16bc7fec..3c90f10f 100644
--- a/third_party/highway/BUILD.gn
+++ b/third_party/highway/BUILD.gn
@@ -6,6 +6,7 @@
 config("libhwy_external_config") {
   include_dirs = [ "src" ]
   defines = []
+  fail(f"target_cpu: {target_cpu}")
   if (target_cpu == "x86") {
     # Highway automatically disables AVX2 and AVX3 targets when compiling in
     # 32-bit mode due to bugs in the generated code. However, when doing so it
@@ -19,7 +20,7 @@ config("libhwy_external_config") {
     # for absl::uint128 is 16.
     defines += [ "HWY_BROKEN_TARGETS=HWY_ALL_SVE" ]
   }
-  if (target_cpu == "ppc64" || target_cpu == "s390x") {
+  if (target_cpu == "ppc" || target_cpu == "s390x") {
     # Not using <asm/hwcap.h> on ppc64/s390x:
     # https://github.com/google/highway/issues/2406
     defines += [ "TOOLCHAIN_MISS_ASM_HWCAP_H" ]
